seesaw/web/session_manager.py

```diff
@@ -43,22 +43,22 @@ def __init__(self, root_dir, save_path, num_cpus_per_session):
     def ready(self):
         return True
 
-    def _new_session(self, task_list):
+    def _new_session(self, task_list, dataset):
```

Comment:
I'm not sure you should keep the dataset variable here, is there a reason why it's kept here?